packages/ui/src/containers/BlockContainer.ts

```diff
@@ -68,7 +72,7 @@ export class BlockContainer {
       return;
     }
     for (let deploy of this.deploys) {
-      const accountKey = deploy.deploy.header.account;
+      const accountKey = deploy.blockHash;
       const balance = await this.balanceService.getAccountBalance(
```

Comment:
`getAccountBalance` require pass in a account key rather than block hash.

Reply:
The response from deploy hash doesn't have account key param.
{
  "deployHash": "deploy1_0fb356b6d76d2f64a9500ed2cf1d3062ffcf03bb837003c8208602c5d3",
  "state": "processed",
  "cost": 11,
  "errorMessage": null,
  "blockHash": "block1_6409191316db2ad075bf005cba502e2a46f83102bceb736356a9c51111"
}

Reply:
Fixed

Reply:
Done